src/Views/AppInfoView.vala

```diff
@@ -115,10 +115,11 @@ namespace AppCenter.Views {
             var oars_header = new Granite.HeaderLabel (_("Content Warnings"));
 
             var oars_flowbox = new Gtk.FlowBox () {
-                halign = Gtk.Align.START,
-                column_spacing = 6,
+                column_spacing = 24,
                 row_spacing = 12,
-                margin_bottom = 24
+                margin_bottom = 24,
+                // Workaround for Flowbox preserving space around for items that don't exist
+                max_children_per_line = 1000
```

Comment:
```suggestion
                orientation = Gtk.Orientation.VERTICAL
```

Reply:
I want it to be horizontal though :sweat_smile:

Reply:
That's how it is :man_shrugging: It depends if you're talking of the widget wrapping which will be vertical or the widget agglomeration with will be horizontal

Reply:
The orientation isn't an issue here. The problem is that FlowBox is reserving `view-width`/`max-children` pixels for each child so if there are less children than `max-children`, it won't `halign` all the way left